### DISCUSSION TITLE
Tiled images

### DISCUSSION THREAD

**Discussion by cubictoast** [Acly/krita-ai-diffusion#495] [General]
Is it possible to enable image tiling somewhere? Checking the settings I can't find an option anywhere.

**Comment by ETCore7** [comment]
Shift-W toggles wrap mode in Krita, then you can try prompts. Some results work well.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/9d901246-c0f4-4591-8e3f-e0c37bc01cb6)

You can also try different models. Digital Artwork XL produces more game-like tiles.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/5c149481-d003-4c80-a9cc-4497e1716794)

It works, but it isn't perfect. Also, the plugin does not see a tiled canvas, so you can't select uneven borders for inpaint smoothing. You'll need to do that manually with a smudge tool.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/62219009-4845-4e37-ab23-78e04b436430)
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/d8b646a0-5fdf-43a2-852f-449784388a5c)

If you get obvious smears you can Gaussian Blur a bit, and then sharpen a couple of times (via Filter in the menu)
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/106464ae-9831-4ee0-a01d-b5309540ebc3)
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/a50cee05-b984-49b6-8a5a-bef65795f8f7)
